frontend/app/quest/Quest_Review.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+    <!-- to do:  update central stylesheet to this style sheet code.-->
+    
+<html ng-app="quest_builder">
```

Comment:
html wrapping tags and the head/body tags are not needed in every partial. If it is being brought into another partial (index.html or app.html), then it can just be the html you need for that view, as it is injected into a valid html document. With this, you are injecting an html document into an html document -- not what we want